### ISSUE TITLE
swe_update() cannot contact GitHub on my computer

### ISSUE THREAD

**Issue by BryanGuillaume** [NISOx-BDI/SwE-toolbox#111]
swe_update() does not seem to work on my computer. Below the message I got on my Matlab:

```
>> swe_update()
Error using swe_update (line 24)
Can't contact GitHub
```
Note that, when typing the url that is in the code (https://github.com/NISOx-BDI/SwE-toolbox/releases/latest) in Safari or Google Chrome, it redirects me to the url https://github.com/NISOx-BDI/SwE-toolbox/releases/tag/v2.1.0.

I am not sure if the issue is only for some users like me (could be linked to a Matlab version or configuration) or if this is more global. @nicholst: Could you please test it on your computer to see if you can reproduce this issue?

Note that the same error appeared for SnPM:

```
>> snpm_update()
Error using snpm_update (line 21)
Can't contact GitHub
```

**Comment by TomMaullin** [comment]
I can confirm that I do not get this error on my computer (just double checked) - I work on windows when I'm using Matlab... perhaps this is an OS thing?

**Comment by BryanGuillaume** [comment]
@TomMaullin: Thank you for the feedback. This issue could indeed be an OS thing or due to a particular version of Matlab as they may have changed the problematic function `urlread` accross Matlab versions. What version of Matlab are you using? On my side, I am currently using Matlab R2017a on a Mac.

**Comment by BryanGuillaume** [comment]
Reading some forums, it seems that this type of issue has occured for some other users of the function `urlread`. This [link](https://nl.mathworks.com/matlabcentral/answers/92506-how-can-i-configure-matlab-to-allow-access-to-self-signed-https-servers) seems to suggest that the issue may come from the fact that Matlab could not authentificate the HTTPS server certificate. Their proposed solution to overcome this issue seems quite cumbersome (need for the user to download the certificate, run a script,...). Thus, I would prefer to avoid this kind of solution. Nevertheless, I have tried to replace the function `urlread`by the function `webread` instead and it seems to  work for me... Thus, I will add a try catch statement in order to use `webread` instead of `urlread` if the latter fails to work.

**Comment by TomMaullin** [comment]
I am running Matlab R2018b. `webread` sounds good. The only reason I used the page with the redirect is to avoid having to update it every time the code is updated.

**Comment by BryanGuillaume** [comment]
This bug has been resolved in PR #115
